src/server/handlers/launchpad.js

```diff
@@ -227,8 +219,11 @@ export const internalGetSnapcraftYaml = async (owner, name, token) => {
           contents: yaml.safeLoad(response.body),
           path
         };
-      } catch (e) {
-        throw new PreparedError(400, RESPONSE_SNAPCRAFT_YAML_PARSE_FAILED);
+      } catch (error) {
```

Comment:
In case of parse error return snapcraft.yaml path (so we can point to it) and error details.